### ISSUE TITLE
Blur applied to notes section

### ISSUE THREAD

**Issue by theuncoder** [devgeeks/Encryptr#193]
Blur currently only applied to the password field but it would be good to have that option open for the notes field as well. I know I would use this feature, but perhaps we need to also ask other users, maybe I'm using my notes for things other users don't.

**Comment by daviddahl** [comment]
:+1:

**Comment by devgeeks** [comment]
Yeah, not sure I would default it that way, but maybe as a setting?

**Comment by jchan-scc** [comment]
yes as a setting
